.golangci.yml

```diff
@@ -57,7 +57,10 @@ linters-settings:
     # Maximum length of variable declaration measured in number of characters, after which linter won't suggest using short syntax.
     max-decl-chars: 50
   gci:
-    local-prefixes: "sigs.k8s.io/cluster-api"
+    sections:
+      - standard
+      - default
+      - prefix(sigs.k8s.io/cluster-api)
```

Comment:
This will block any of the cluster-api related imports from being declared before 3rd party libraries.